components/sceneutil/riggeometry.cpp

```diff
@@ -145,7 +146,7 @@ bool RigGeometry::initFromParentSkeleton(osg::NodeVisitor* nv)
             continue;
         }
 
-        mBoneSphereMap[bone] = it->second.mBoundSphere;
+        mBoneSphereVector.push_back(std::make_pair(bone, it->second.mBoundSphere));
```

Comment:
Use emplace_back instead, it allows to avoid a copy.

Reply:
Done.